stages/org.osbuild.mkfs.ext4

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+"""
+Construct an ext4 file-system via mkfs.ext4(8)
+
+Construct a ext4 file-system with the given options at the device
+specified via `device`.
+
+Buildhost commands used: `mke2fs`.
+"""
+
+import subprocess
+import sys
+
+
+import osbuild.api
+
+
+SCHEMA_2 = r"""
+"devices": {
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["device"],
+  "properties": {
+    "device": {
+      "type": "object",
+      "additionalProperties": true
+    }
+  }
+},
+"options": {
+  "additionalProperties": false,
+  "required": ["uuid"],
+  "properties": {
+    "uuid": {
+      "description": "Volume identifier",
+      "type": "string"
+    },
+    "label": {
+      "description": "Label for the file system",
+      "type": "string"
+    }
+  }
+}
+"""
+
+
+def main(devices, options):
+    device = devices["device"]["path"]
```

Comment:
Is `path` required by the schema?

Reply:
`devices`, `input`, `mounts` in the arguments are the returned values from the host services (not data that comes directly from the manifest). Maybe it is a good point to assert that the individual host services return replies that include the path entry.